sharktank_models/llama3.1/test_llama.py

```diff
@@ -0,0 +1,111 @@
+import iree.compiler
+import iree.runtime
+import numpy
+import os
+import pytest
+import torch
+
+page_size = 12288
+block_size = 16
+
+
+@pytest.fixture
+def llama_mlir():
+    dirname = os.path.dirname(__file__)
+    return f"{dirname}/assets/toy_llama.mlir"
```

Comment:
Could add a comment in this file or a README at `sharktank_models/llama3.1` explaining what about this model is a "toy" (number of layers, properties of the weights, etc.) and how to regenerate it

Reply:
Added a readme comment.